.php_cs.dist

```diff
@@ -17,26 +17,30 @@ return \PhpCsFixer\Config::create()
     ->setRiskyAllowed(true)
     ->setRules([
         '@Symfony' => true,
+        '@Symfony:risky' => true,
         '@PHP70Migration' => true,
+        '@PHP70Migration:risky' => true,
 
         'concat_space' => ['spacing' => 'one'],
+        'declare_strict_types' => false,
```

Comment:
could you please explain why `false`?

Reply:
sure I can !
rule is not enabled on master, `@php7:risky` has it by default on, so I turned if off.
it is conflicting with header rule, ending up adding 2 headers (above and below declare statement), for that I didn't wanted to fix everything in single PR ;)

Reply:
this is what @keradus is talking about:

```diff
<?php
/**
 * Copyright © 2017-2018 Maks Rafalko
 *
 * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
 */

declare(strict_types=1);
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
```

Is it a bug of PHP-CS-Fixer?

Reply:
this is known conflict between those 2 rules, we have raised issue for it for a while, nobody helped solving it so far